docs/examples/ca-issuer.yaml

```diff
@@ -4,5 +4,4 @@ metadata:
   name: ca-issuer
 spec:
   ca:
-    secretRef:
-      name: ca-key-pair
+    secretName: ca-key-pair
```

Comment:
What are the default keys are used for the key and cert?

Reply:
These default to the standard Kubernetes Certificate key names, e.g. `tls.crt` and `tls.key` (similar to the `certificate.spec.secretName` field).

This is done so that in future, we can eventually chain together issuers (e.g. one issuer produces a certificate that can be used for signing, and another issuer then consumes that keypair as it's source for signing)